src/Asana/AsanaApiClient.php

```diff
@@ -75,7 +74,7 @@ public function configureOptions(OptionsResolver $resolver): void
             'yesplan_insaleDate',
             'yesplan_percent',
             'yesplan_status',
-            'yesplan_profile'
+            'yesplan_profile'    
```

Comment:
Remember to apply coding standards. And if the coding standards allow trailing whitespace, they should be updated.

Reply:
I forgot about running the coding stadards, I have done so now...